engine/src/main/java/com/redhat/cloud/notifications/processors/email/EmailSender.java

```diff
@@ -242,9 +242,6 @@ protected Email buildEmail(Set<User> recipients, String subject, String body) {
         Set<String> usersEmail = recipients.stream().map(User::getUsername).collect(Collectors.toSet());
         Email email = new Email();
         email.setBodyType(BODY_TYPE_HTML);
-        if (featureFlipper.isAddDefaultRecipientOnSingleEmail()) {
-            email.setRecipients(Set.of(noReplyEmail));
-        }
```

Comment:
Let's keep it in the `engine` for now. When I spoke about removing the noreply email, I only meant to do that in the connector :smiley:

Reply:
Thanks but you reverted it in the wrong method :smile:

Reply:
Wow, sorry about that. Now it's back to what it was before :sweat: